BUILD

```diff
@@ -1 +1 @@
-python_requirements()
+python_requirements(name="reqs")
```

Comment:
I couldn't find this in the changelog, but after updating to 2.13 pants started complaining that names are required for root-level targets:

```
$ ./pants dependencies ::
14:19:33.73 [ERROR] 1 Exception encountered:

  MappingError: Failed to parse ./BUILD:
Targets in root-level BUILD files must be named explicitly.
```